packages/grafana-data/src/valueFormats/dateTimeFormatters.ts

```diff
@@ -370,18 +370,13 @@ export const dateTimeAsIsoNoDateIfToday = toDateTimeValueFormatter('YYYY-MM-DD H
 export const dateTimeAsUS = toDateTimeValueFormatter('MM/DD/YYYY h:mm:ss a');
 export const dateTimeAsUSNoDateIfToday = toDateTimeValueFormatter('MM/DD/YYYY h:mm:ss a', 'h:mm:ss a');
 
-export function getDateTimeAsLocalFormat() {
-  return toDateTimeValueFormatter(
-    localTimeFormat({
-      year: 'numeric',
-      month: '2-digit',
-      day: '2-digit',
-      hour: '2-digit',
-      minute: '2-digit',
-      second: '2-digit',
-    })
-  );
-}
```

Comment:
this was done in a function for a reason, as the local stuff access Intl api which is not available from node environments

Reply:
Makes sense I have updated the PR.